pytest-backend/README.md

```diff
@@ -0,0 +1,15 @@
+# pytest-exasol-backend Plugin
+
+Content to be provided
+
+## Features
+
+Content to be provided
```

Comment:
Please add some initial documentation.

Reply:
Yes sure. I planned to do this. Just wanted to get the feedback for the solution first, so I don't have to rewrite the README.md.